content/reference/admin/private_locations/private_packages/index.md

```diff
@@ -117,6 +117,32 @@ This configuration includes the following parameters:
 - **bucket**: The name of the bucket where packages are uploaded to on GCP Cloud Storage.
 - **path:** The path of a folder in Cloud Storage bucket. (optional)
 
+#### Filesystem Storage
+This option allows the storage of simulations directly on the control-plane filesystem.
+```bash
+  repository = {
+    type = "filesystem"
+    directory = "/data/gatling-repository"
+    upload {
+      # optional, default: /tmp
+      # location to temporarily store your incoming simulation during the upload process
+      directory = "/tmp"
```

Comment:
```suggestion
      # Directory to temporarily store your incoming simulation during the upload process (optional, default: /tmp)
      directory = "/tmp"
```

Reply:
Consistency with other doc optional fields + use directory instead of location to match key?